include/silo/storage/column/float_column.h

```diff
@@ -0,0 +1,33 @@
+#ifndef SILO_FLOAT_COLUMN_H
+#define SILO_FLOAT_COLUMN_H
+
+#include <string>
+#include <vector>
+
+#include "silo/storage/column/column.h"
+
+namespace silo::storage::column {
+
+class FloatColumn : public Column {
+  public:
+   template <class Archive>
+   [[maybe_unused]] void serialize(Archive& archive, const unsigned int /* version */) {
+      archive& values;
+   }
+
+  private:
+   std::vector<double> values;
+
+  public:
```

Comment:
Only a style thing: Do we want to have multiple sections of public and private in the class?

Reply:
In general probably not, but in this case I consider the Archive thing boiler plate, that's simply on top of the class definition. 
Then we have the rest with a private and a public section.

We could change it, but that's how we have it everywhere AFAIK.

Reply:
I also think the grouping by private public is a good idea in general, but other groupings can outweigh that guideline such as Fabian's mentioned consideration